### ISSUE TITLE
"Back to Top" link focus indicator not discernible against dark background

### ISSUE THREAD

**Issue by SaraSoueidan** [w3c/aria-practices#2339]
The "Back to Top" link’s focus indicator becomes hard to discern when the link is positioned atop of the dark footer. The color contrast is too low and does not meet WCAG requirements.

Not focused:
<img width="193" alt="Screen Shot 2022-05-20 at 17 57 56" src="https://user-images.githubusercontent.com/2527933/169555904-e7ee5b99-a698-4622-bf75-5cf0a47f5669.png">

Focused:
<img width="186" alt="Screen Shot 2022-05-20 at 17 58 07" src="https://user-images.githubusercontent.com/2527933/169555951-9cba9015-49db-4043-96a5-bfa011bbdf79.png">

Contrast ratio:
<img width="592" alt="Screen Shot 2022-05-20 at 18 00 47" src="https://user-images.githubusercontent.com/2527933/169556425-d0b8a3a2-e0c4-4adb-9659-74ee1e3c35f9.png">

It looks fine when it receives focus with the white background behind it. 

<img width="189" alt="Screen Shot 2022-05-20 at 17 58 28" src="https://user-images.githubusercontent.com/2527933/169556031-260b9cd3-5929-43b2-9b3a-07d5a6af5e08.png">

Then again, the link only receives keyboard focus after tabbing through the page’s links all the way down to the footer. As such, it will almost always be positioned above the footer when it does receive focus.

The only way I could tell where my focus was when I tabbed to it was via the URL shown in the status bar.

A possible way to avoid this is to use a "double outline" (one light outline, and one dark outline) when it receives focus, so that it stands out on both light and dark backgrounds.
